Solution1/Solution1.Tests/ConsoleApp/Services/UserCommunicateServiceTests.cs

```diff
@@ -1,97 +1,133 @@
 ﻿using BusinessLayer.DTOs;
 using BusinessLayer.Services.Abstract;
+using ConsoleApp.Command;
+using ConsoleApp.Command.Abstract;
 using ConsoleApp.Services;
 using Microsoft.Extensions.Logging;
 using Moq;
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
 using Xunit;
+using FluentValidation;
+using FluentValidation.Results;
+using ConsoleApp.Services.Abstract;
+using Weather.Tests.Infrastructure;
 
 namespace Weather.Tests.ConsoleApp.Services
 {
     public class UserCommunicateServiceTests
     {
-        private readonly UserCommunicateService _userCommunicationService;
-        private readonly Mock<IWeatherServiсe> _weatherServiceMock;
+        private readonly IUserCommunicateService _userCommunicationService;
```

Comment:
UserCommunicateService - you test exact implementation, not all implementations will behave exact way as you test

Reply:
+